feature-libs/asm/components/customer-selection/customer-selection.component.html

```diff
@@ -24,60 +27,168 @@
       *cxFeature="'!formErrorsDescriptiveMessages'"
       [control]="customerSelectionForm.get('searchTerm')"
     ></cx-form-errors>
+
+    <div *ngIf="searchResults | async as results" #resultList>
+      <div
+        *ngIf="
+          !(searchResultsLoading$ | async) &&
+          searchTerm.value.length >= 3 &&
+          searchByCustomer &&
+          !!results.entries &&
+          results.entries.length <= 0
+        "
+        class="cx-message cx-message-info"
+        ng-reflect-ng-class="[object Object]"
+      >
```

Comment:
suggest add a method which is more readable and can write unit test for your logic
```
isXXXVisible(): boolean {
  return !(this.searchResultsLoading$ | async) &&
         this.searchTerm.value.length >= 3 &&
         this.searchByCustomer &&
         !!this.results.entries &&
         this.results.entries.length <= 0;
}
```